src/object_storage/objects.rs

```diff
@@ -145,6 +145,11 @@ impl Object {
         name: ref String
     }
 
+    transparent_property! {
+        #[doc = "Object hash or ETag."]
```

Comment:
Maybe mention md5 explicitly? It seems to be a part of the API contract.

Reply:
I tryed something...